charms/argo-controller/src/prometheus_alert_rules/KubeflowArgoControllerServices.rules

```diff
@@ -0,0 +1,24 @@
+groups:
+- name: KubeflowArgoControllerServices
+  rules:
+  - alert: KubeflowServiceDown
+    expr: up{} < 1
+    for: 5m
+    labels:
+      severity: critical
+    annotations:
+      summary: "{{ $labels.juju_charm }} service is Down ({{ $labels.juju_model }}/{{ $labels.juju_unit }})"
+      description: |
+       One or more targets of {{ $labels.juju_charm }} charm are down on unit {{ $labels.juju_model }}/{{ $labels.juju_unit }}.
```

Comment:
Is this description correct ? I believe this alert rule fires if less then 1  target is up right ?

Reply:
The up represent the state of metrics scraping. I'll try to explain it with example:
For charm with two target, we can have three scenarios:
- both targets are working fine -> up{juju_charm="my-charm"} == 1
- one target is up and one down -> up{juju_charm="my-charm"} == 0
- both targets are down -> up{juju_charm="my-charm"} == 0